builtin/credential/okta/backend.go

```diff
@@ -57,7 +62,7 @@ type backend struct {
 	*framework.Backend
 }
 
-func (b *backend) Login(ctx context.Context, req *logical.Request, username string, password string) ([]string, *logical.Response, []string, error) {
+func (b *backend) Login(ctx context.Context, req *logical.Request, username, password, totp string) ([]string, *logical.Response, []string, error) {
```

Comment:
Returning 4 parameters is a code smell. I realize it's a bit outside the context of this change, but can this be refactored to return fewer values? Barring that, returning them in a response object type? At the minimum, labeling the return values would help.